README.adoc

```diff
@@ -89,26 +89,24 @@ If you want to build everything, use the `build` task:
 	$ ./gradlew build
 ----
 
-
-
 == Modules
-There are a number of modules in Spring Boot, here is a quick overview:
+There are several modules in Spring Boot; here is a quick overview:
 
 
 
 === spring-boot
-The main library providing features that support the other parts of Spring Boot, these include:
+The main library providing features that support the other parts of Spring Boot; these include:
```

Comment:
Make two sentences to remove the semicolon. (You are right to replace the comma splice, but we favor a different replacement.)